components/accept-financial-contributions/ContributionCategoryPicker.js

```diff
@@ -153,25 +139,15 @@ class ContributionCategoryPicker extends React.Component {
                 width={[null, 280, 312]}
               >
                 <Flex flexDirection="column" justifyContent="center" alignItems="center">
-                  <Image src={skateboardingIllustration} alt={intl.formatMessage(this.messages.host)} />
+                  <Image src={acceptHostIllustration} alt={intl.formatMessage(this.messages.host)} />
                   <Link
                     route="accept-financial-contributions"
                     params={{
                       slug: router.query.slug,
                       path: 'host',
                     }}
                   >
-                    <StyledButton
-                      fontSize="13px"
-                      buttonStyle="dark"
-                      minHeight="36px"
-                      mt={[2, 3]}
-                      mb={3}
-                      px={3}
-                      onClick={() => {
-                        this.props.onChange('path', 'host');
-                      }}
-                    >
+                    <StyledButton fontSize="13px" buttonStyle="dark" minHeight="36px" mt={[2, 3]} mb={3} px={3}>
```

Comment:
**[:ice_cream: Details & personal preferences]** When you want a button to have a minimum width, it's better to use `minWidth` for two reasons:
1. When translated, the button labels usually have different sizes. Using padding can create strange results when the translated string is very long.
2. It will make sure the button keeps its size if it ever takes the `loading `state

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
That's a good idea, and I did notice that the padding makes it kind of weird sometimes. What do you suggest as a good `minWidth`? Just something that makes all the buttons look relatively the same size?

Reply:
> Just something that makes all the buttons look relatively the same size?

Yes that's good enough. I often put the width of the button in figma as `minWidth`